src/ca/team2706/vision/trackerboxreloaded/ExampleServer.java

```diff
@@ -3,43 +3,39 @@
 import ca.team2706.vision.trackerboxreloaded.Main.VisionData;
 import edu.wpi.first.networktables.NetworkTable;
 import edu.wpi.first.networktables.NetworkTableInstance;
-import edu.wpi.first.networktables.NetworkTableValue;
 
 public class ExampleServer {
 	/**
 	 * the vision table
 	 */
 	public static NetworkTable vision;
-	
-	/**
-	 * the fps table
-	 */
-	public static NetworkTable fps;
-	
-	/**
-	 * the data table
-	 */
-	public static NetworkTable data;
 
-	
-	@SuppressWarnings("unused")
 	public static void main(String[] args) {
-		//Inits stuff
+		// Must be included!
+		System.loadLibrary("opencv_java310");
+		
+		// Inits stuff
 		NetworkTableInstance instance = NetworkTableInstance.getDefault();
 		instance.setUpdateRate(0.02);
 		instance.startServer();
 		vision = instance.getTable("vision");
-		data = vision.getSubTable("data");
-		fps = vision.getSubTable("fps");
 		while (true) {
-			VisionData visionData = Main.VisionData.decode(data);
-			NetworkTableValue fpsValue = fps.getEntry("fps").getValue();
-			if(fpsValue.isDouble()){
-				System.out.println("fps: "+fpsValue.getDouble());
-			}
 			try {
-				Thread.sleep(1);
-			} catch (InterruptedException e) {
+				VisionData visionData = Main.VisionData.decode(vision);
+				System.out.println();
+				System.out.println();
+				System.out.println();
+				System.out.println();
+				System.out.println();
+				System.out.println("ctrX: "+visionData.ctrX);
+				System.out.println("numTargetsFound: "+visionData.numTargetsFound);
+				System.out.println("fps: "+visionData.fps);
+				try {
+					Thread.sleep(10);
+				} catch (InterruptedException e) {
+					e.printStackTrace();
+				}
+			} catch (Exception e) {
 				e.printStackTrace();
 			}
 		}
```

Comment:
I thought I renamed this file yesterday?

Reply:
That was on Matt's branch I think.
At 230 today I will  fix all da conflicts

Reply:
Ok, that makes sense. I suggest you:

`matt_math` --merge--> `NetworkTables`
fix conflicts, build, test
`NetworkTables` --pull request--> `matt_math`